docs/UserGuide.md

```diff
@@ -947,16 +949,16 @@ The information below specifies parameter description, constraints and usage thr
-| i/        | `edit-i-role`, `delete-i-role` | Index in internship role list                                                      | Positive integer less than or equal to 2147483649                                                                  | 1, 2147483647                                         | -1, 2147483649     |
+| Parameter | Command                        | Description                                                                        | Constraints                                                                                                                      | Valid Examples                                        | Invalid Examples   |
+|-----------|--------------------------------|------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------|--------------------|
+| n/        | `add-i-role`                   | Name of the company you are applying to                                            | NA                                                                                                                               | Google, Tiktok                                        | NA                 |
+| r/        | `add-i-role`                   | Name of the role you are applying for                                              | NA                                                                                                                               | Software Engineer, Backend Engineer                   | NA                 |
+| c/        | `add-i-role`                   | Cycle of the internship you are applying for                                       | NA                                                                                                                               | Winter 2023, Summer 2024, Spring                      | NA                 |
+| d/        | `add-i-role`                   | Description of the internship role (optional parameter)                            | NA                                                                                                                               | Very important internship                             | NA                 |
+| p/        | `add-i-role`                   | Pay of the internship role (optional parameter)                                    | Only numbers with up to one decimal point allowed. Must be a non-negative number. Display will be formatted to 2 decimal places. | 1000, 1000.00, 1000.000000 (will be shown as 1000.00) | $1000, -1, -0      |
```

Comment:
Display will truncate pay to 2 decimal places.
Also as a side-note, are we storing more than 2 DP? If not it might be more accurate to say that input will be truncated instead of display will truncate